tests/org.eclipse.n4js.hlc.tests/src/org/eclipse/n4js/hlc/tests/N4jscBasicTest.java

```diff
@@ -51,7 +51,7 @@ public void deleteWorkspace() throws IOException {
 	@Test
 	public void testMainArgsCompileAllKeepCompiling() {
 		CliCompileResult cliResult = n4jsc(COMPILE(workspace));
-		assertEquals(cliResult.toString(), 19, cliResult.getTranspiledFilesCount());
+		assertEquals(cliResult.toString(), 16, cliResult.getTranspiledFilesCount());
```

Comment:
Why did the number of compiled files go down?

Reply:
This number went down because already existing js files in src-gen folders are not counted anymore, which are `trash.js`, `garbage1.js` and `garbage2.js`.